lib/blockscore/base.rb

```diff
@@ -1,10 +1,12 @@
 require 'blockscore/connection'
+require 'uri'
 
 module BlockScore
   class Base
     extend Connection
 
     ABSTRACT_WARNING = 'Base is an abstract class, not an API resource'.freeze
+    API_URL = URI.parse('https://api.blockscore.com/').freeze
```

Comment:
I was cautioned not to include other gems, so I chose to use what was readily available and backwards compatible with the Ruby version supported. The more dependencies on other gems the more possibility also of issues with Ruby version support conflicts. After I looked at `Addressable::URI` I saw no distinct advantage.